katas/yatzy/index.js

```diff
@@ -0,0 +1,3 @@
+function yatzy(){}
```

Comment:
slight nitpick. in the refactoring code, the function for yatzy is spelled with a capital Y. Should we change for consistency?